src/gap.c

```diff
@@ -1134,32 +1134,32 @@ static int SetExitValue(Obj code)
 
 /****************************************************************************
 **
-*F  FuncGAP_EXIT_CODE() . . . . . . . . Set the code with which GAP exits.
+*F  FuncGapExitCode() . . . . . . . . Set the code with which GAP exits.
 **
 */
 
-static Obj FuncGAP_EXIT_CODE(Obj self, Obj code)
+static Obj FuncGapExitCode(Obj self, Obj code)
```

Comment:
So tangentially related to this PR (which BTW needs to be rebased, then it could be merged): I just had need to query the exit code, and there simply is no way to do this. However, the natural name for a function to do this is taken... Can we either rename this to `SetGapExitCode` (and then later we can can add `GapExitCode` or `GetGapExitCode` to query it); or else change this function to return the exit code, and then also to allow calling it with 0 args, to just get the exit code w/o changing it?